frontend/simple/assets/sass/_groupincome.scss

```diff
@@ -16,6 +16,36 @@
   font-weight: 700;
 }
 
+body {
+  font-size: $size-6; // Bulma doesnt have a variable
+}
+
+// ------ index.html ------ //
+// QUESTION: This doesnt look like the best place
+// to style the index.html / main.js
+// Why isn't that file a vue file like all the others?
```

Comment:
`index.html` and `main.js` cannot be `.vue` files because `.vue` files exist solely for the purpose of getting compiled into `.js` file(s) that are then loaded via the main js bundle.

In other words, `index.html` and `main.js` are bootstrapping files for the whole system, and `.vue` files are components that get compiled into a javascript bundle.

Basically, it's just because of the way that JavaScript, HTML, and VueJS work that it's this way.

Reply:
Hum... I see... Ok